exercises/gigasecond/description.md

```diff
@@ -1,3 +1,4 @@
-Calculate the moment when someone has lived for 10^9 seconds.
+Given a `start` value, write a function that returns what the date will
```

Comment:
My suggestion, would be:

    Given a moment, determine the moment that would be after a gigasecond (10^9 seconds) has passed.

@petertseng suggestion as converted with "no objection here" from him:

    Given a moment as input, determine the moment that is 10^9 seconds past that moment.